### ISSUE TITLE
Display user friendly error if running against Synapse

### ISSUE THREAD

**Issue by dagguh** [matrix-org/cerulean#43]
**Describe the bug**
Threads are empty.

**To Reproduce**
1. Go to [a thread](https://cerulean.matrix.org/@matthew:dendrite.matrix.org/!o3r2XygCOPz3cWID:dendrite.matrix.org/$K94YqDqDTfv-DsHYQAdZL1BsUjWzlwqr5bVrXIeyxZs)
2. See empty page

**Expected behavior**
I'd expect to see the thread contents. Or to understand why I'm not able to see them. Or even what can I do to fix that.

**Screenshots**
The page is empty.
Also, the Firefox console logs an error:`{ errcode: "M_UNRECOGNIZED", error: "Unrecognized request" }`:
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/1194704/104137796-c4b47b80-539f-11eb-8905-1f98a027f9ea.png">

It's a body of a HTTP 400 response to a POST https://matrix.org/_matrix/client/unstable/event_relationships

 - OS: iOS
 - Browser Firefox 84.0.1

**Additional context**
Logged in via federation to `matrix.org`.

**Comment by Half-Shot** [comment]
Cerulean does not support Synapse (which matrix.org runs) because it doesn't support some of the MSC implementations used (as seen in the wiki). You'll either have to use dendrite.matrix.org, or run your own Dendrite server with the MSCs enabled in the config.

@Kegsay do you want to keep this issue open because the application isn't showing an error, or should this be closed?

**Comment by dagguh** [comment]
Would be nice to see a message like:
> Your homeserver matrix.org doesn't support XYZ.

**Comment by kegsay** [comment]
Yes, the bug here is that we should display this coherently to the user that it no worky.

**Comment by Half-Shot** [comment]
Relatedly we could get Cerulean to determine this dynamically if Dendrite implemented https://github.com/matrix-org/dendrite/issues/1696

**Comment by kegsay** [comment]
^ is the right approach here.
